### DISCUSSION TITLE
Più account amministratore

### DISCUSSION THREAD

**Discussion by luigirotondi** [iisgiua/giuaschool#701] [Idee e proposte]
Salve! Prossimamente sarà possibile aggiungere più di un account amministratore?

**Comment by trinko** [comment]
La segno come richiesta.

**Comment by sempliciniluigi89** [comment]
Ma è già possibile farlo, basta accedere nella sezione utenti in phpMyAdmin
e sul ruolo dell'utente  invece di lasciare Doc=docente etc scrivere AMM e
l'utente diventa automaticamente amministratore

Il Gio 16 Mag 2024, 23:22 Antonello Dessì ***@***.***> ha
scritto:

**Reply by luigirotondi2009** [reply]
Dopo aver eseguito l'accesso al PhpMyAdmin dove devo cliccare?

**Comment by sempliciniluigi89** [comment]
sul menu a tendina che esce a sinistra con tutte le tabelle ,cerchi utenti
e da lì si clicca sull'utente per il quale si intende cambiare/modificare
qualcosa ,scendendo in basso trovate "ruolo" e da lì inserite AMM

Il Ven 17 Mag 2024, 15:18 luigirotondi2009 ***@***.***> ha
scritto:
